### PR TITLE
Add save distance funtion for persistence when a new stadium and dist…

### PR DESCRIPTION
**Pull request by Roger-Chavez**
Closes #(#56 )

### Key features
* Distance persistence -  added a static save function for distance data
* updated csv files to only contain stadium id's and distance
### Future Improvements
* none

### Notes
none

### CODE REVIEW

File: src/datastore/DistanceBetweenStadiums.csv [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
-60,Minute Maid Park,76,Tropicana Field,790
-79,Nationals Park,52,Oriole Park at Camden Yards,50
-79,Nationals Park,71,PNC Park,195
-79,Nationals Park,51,Sun Trust Park,560
-71,PNC Park,78,Rogers Centre,225
-71,PNC Park,57,Progressive Field,115
-71,PNC Park,68,Yankee Stadium,315
-78,Rogers Centre,74,Safeco Field,2070
-74,Safeco Field,66,Target Field,1390
+73,74,680
```
Comment (ChrisRzech) [inline src/datastore/DistanceBetweenStadiums.csv:2]: Comment go bye bye?

File: src/datastore/database.cpp [hunk 2]
```diff
@@ -182,6 +182,30 @@ void Database::loadDistancesFromFile(const std::string& filepath)
...
+        columns.push_back(std::to_string(std::get<0>(distance)));
+        columns.push_back(std::to_string(std::get<1>(distance)));
+        columns.push_back(std::to_string(std::get<2>(distance)));
+        allRows.push_back(columns);
+    }
+    // Call saveData to store data at the path provided
+    saveData(path, allRows);
+}
+
+
```
Comment (ChrisRzech) [inline src/datastore/database.cpp]: :pepeHmm: 2 empty lines i see

**Review by ChrisRzech — CHANGES_REQUESTED** [review]
Wuh oh

**Review by ChrisRzech — APPROVED** [review]
👍 assuming the changes are made so i can go sleepy-bye

**Review by Roger-Chavez — COMMENTED** [review]
Looks good 👍🏽

**Review by WakkaTrout — APPROVED** [review]
I don't see any problems... so approved

**Pull request merged**